programs-remote-listings/templates/shortcode-programs-single.php

```diff
@@ -88,7 +93,10 @@
 
         <?php if ($rs_the_program->teacher_details->teacher_objects) : ?>
             <div class="rs-teachers-container">
-                <h2 class="rs-teachers-title"><?php echo _n('Teacher', 'Teachers',
+                <?php $teacher_settings = $rs_the_program->teacher_details->teacher_settings; ?>
```

Comment:
I'm a bit paranoid about warnings on customer sites, but this is likely fine as we would hopefully be very careful about removing or renaming any properties within our API results.